radicle-cli/src/commands/issue.rs

```diff
@@ -235,7 +235,14 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         Operation::List => {
             for result in issues.all()? {
                 let (id, issue, _) = result?;
-                println!("{} {}", id, issue.title());
+                let mut line = format!("{} \"{}\"", id, issue.title().escape_default());
+
+                let assigned: Vec<String> = issue.assigned().map(|p| p.to_string()).collect();
+                if !assigned.is_empty() {
+                    line = format!("{} {}", line, &assigned.join(", "));
+                }
```

Comment:
It's clearer if you set `line` only once and keep it immutable, eg. `let line = if assigned { ... } else { ... };`